docs/terms/contestation-period.md

```diff
@@ -0,0 +1,10 @@
+---
+id: contestation-period
+title: Contestation Period
+type: concept
+typeid: term
+hoverText: "The time window between closing the head and `fan out` allowed to `contest`."
+glossaryText: "The time window between closing the [head](../terms/hydra-head) and `fan out` allowed to `contest`."
```

Comment:
Seems like this is necessary to have the explanation in the glossary. So this duplication seems to be required :(

Reply:
yes, `hoverText`, `glossaryText` and the body of the term are required for a specific feature, so there is no way to replace one with another to remove redundancy

Reply:
for me the `terms` replacements to get hover does work.

Reply:
if we are not interested in the plugin features, then yes we could write a glossary.md ourselves manually.
I used this approach as it is standard in docusaurus.

Reply:
> for me the terms replacements to get hover does work.
You mean things like `%%contestation period|contestation-period%%`? That was not working for me.

> it is standard in docusaurus.
It's an existing plugin, yes. But it's doing more than we need, doesn't work right now (the thing above) and requires us to duplicate work -> I would like to keep things simple.

If we don't have the term replacement with links into the glossary, a manual glossary page provides the same value, with much simpler configuration (none).

Reply:
plugin removed and a single glossary page is provided